boards/arm/curie_ble/board.cmake

```diff
@@ -0,0 +1,13 @@
+if(DEFINED ENV{ZEPHYR_FLASH_OVER_DFU})
```

Comment:
This should be removed. We have some legacy environment variables, but those are for legacy purposes.

Nowadays, users can just run `cmake -DBOARD_FLASH_RUNNER=dfu-util` to set the default flasher, or run `west flash -r dfu-util` to pick a flash runner at runtime.

Reply:
if I understand you correctly, lines L:1...L:5 should be removed?

Reply:
Lines 1 through 8 inclusive should be removed. The arduino_101_sss and arduino_101 files have been grandfathered in from a time when we were using Kbuild and had a bunch of shell scripts to do the flashing; they're really not good examples.